### PR TITLE
Initialize collector on build to improve container start time

### PR DESCRIPTION
**Pull request by vtols**

### CODE REVIEW

**Comment by rodionos** [comment]
I'd like to review a list of files in the /opt/axibase-collector directory that were changed after the first start and after logs were removed. My expectation is that it will be only files in the acdb directory (Derby db).

**Comment by vtols** [comment]
[List of files changing on first start](https://github.com/axibase/docker-axibase-collector/files/1340938/diff.txt)

**Comment by vtols** [comment]
[List](https://github.com/axibase/docker-axibase-collector/files/1340992/diff.txt) of files changed after logs removed and susequent start

**Comment by rodionos** [comment]
Lets also delete conf/keystores/client.keystore. It will be recreated.
Also, we need to run some clean-up SQL queries (they will be posted into 4594).

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,9 @@ RUN apt-get update && apt-get install --no-install-recommends -y openjdk-8-jre w
     && tar -xzvf axibase-collector-*.tar.gz -C /opt/ && rm axibase-collector-*.tar.gz \
     && mkdir -p /opt/axibase-collector/exploded/webapp \
     && unzip /opt/axibase-collector/lib/axibase-collector.war -d /opt/axibase-collector/exploded/webapp \
-    && mv /tmp/entrypoint.sh /opt/axibase-collector/bin/
+    && mv /tmp/entrypoint.sh /opt/axibase-collector/bin/ \
+    && /opt/axibase-collector/bin/start-collector.sh && /opt/axibase-collector/bin/stop-collector.sh \
```
Comment (simakmi) [inline Dockerfile]: Collector started in background mode so if you execute stop script after start script - collector will be interrupted before initialization completed

Comment (rodionos) [reply Dockerfile]: Good point. The script needs to wait until the collector has completed the initialization process.

Comment (simakmi) [reply Dockerfile]: I forgot that I have fixed the start script and added the wait until the collector has completed the initialization process.

Comment (vtols) [reply Dockerfile]:
```text
 * [Collector] Waiting for Collector to start, pid=4412 ...
 * [Collector] Checking Collector web-interface port 9443 ...
 * [Collector] Waiting for Collector to bind to port 9443 ...( 1 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 2 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 3 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 4 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 5 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 6 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 7 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 8 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 9 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 10 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 11 of 30 )
 * [Collector] Waiting for Collector to bind to port 9443 ...( 12 of 30 )
 * [Collector] Collector web interface:
 * [Collector] https://172.17.0.4:9443
 * [Collector] https://127.0.0.1:9443
 * [Collector] Collector start completed.
 * [Collector] Stopping ...
 * [Collector] Waiting for Collector to stop, pid=4412 .
 * [Collector] Collector stop completed
```

**Pull request merged**